src/libc_installation.zig

```diff
@@ -182,7 +184,19 @@ pub const LibCInstallation = struct {
         var self: LibCInstallation = .{};
 
         if (is_darwin) {
-            @panic("Darwin is handled separately via std.zig.system.darwin module");
+            if (!std.zig.system.darwin.isSdkInstalled(args.allocator))
+                return error.DarwinSdkNotFound;
+            const sdk = std.zig.system.darwin.getSdk(args.allocator, args.target) orelse
+                return error.DarwinSdkNotFound;
+            defer args.allocator.free(sdk.path);
+
+            self.include_dir = try fs.path.join(args.allocator, &.{
+                sdk.path, "usr/include",
+            });
+            self.sys_include_dir = try fs.path.join(args.allocator, &.{
+                sdk.path, "usr/include",
```

Comment:
I take it we do not want to make framework dir part of `LibCInstallation`?

Reply:
It effectively is, since it can always be found at `../../System/Library/Frameworks` which is exactly what `detectLibCFromLibCInstallation` does. Thankfully the Darwin SDK paths are reasonable, and this property holds. On other operating systems such as Linux, the directories are all over the place, so we have to store multiple different named path roots.